CONTRIBUTING.md

```diff
@@ -88,27 +88,25 @@ Sygnal follows the [Synapse code style].
 Many of the conventions are enforced by scripts which are run as part of the
 [continuous integration system](#continuous-integration-and-testing).
 
-To help check and fix adherence to the code style, you can run `scripts-dev/lint.sh`
-locally. You'll need Python 3.7 or later, and to install a number of tools:
+To help check and fix adherence to the code style, you can run `tox`
+locally. You'll need Python 3.7 or later, and a virtual environment configured and
+active:
 
-```
-# Install the dependencies
-pip install -U black flake8 isort mypy mypy-zope
+```bash
+# Activate the virtual environment
+source ./venv/bin/activate
 
-# Run the linter script
-./scripts-dev/lint.sh
+# Run the code style check
+tox -e check_codestyle
+
+# Run the types check
+tox -e check_codestyle
 ```
 
 **Note that the script does not just test/check, but also reformats code, so you
 may wish to ensure any new code is committed first**.
 
-By default, this script checks all files and can take some time; if you alter
-only certain files, you might wish to specify paths as arguments to reduce the
-run-time:
-
-```
-./scripts-dev/lint.sh path/to/file1.py path/to/file2.py path/to/folder
-```
+These commands will consider the paths and files related to the project (i.e. everything in `sygnal/` and in `tests/` as well as the `setup.py` file).
```

Comment:
I'm not sure _why_ we do this, but it seems like the rest of the file has manual line-wrapping at ~78 characters... Could you please break this line to match the rest?